src/main/kotlin/org/acme/findValue/FindValue.kt

```diff
@@ -11,30 +11,31 @@ import javax.ws.rs.core.MediaType
 @ApplicationScoped
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
-class FindValue (@Inject private val config: Config) {
+class FindValue(@Inject private val config: Config) {
 
+    val regex: Regex = "\\b${config.delimiter}\\b".toRegex()
```

Comment:
Regexp expression in my implementation depends on current value what need to find. For Example, when I want to find number 666 I use regexp - "\b666\b". \b - is not a space symbol, it is a word boundary (same as "not \w"). 

It's a first step, next step I use static regexp to split text into numbers. 
Your commit is too early. =)